pkg/policy/api.go

```diff
@@ -359,6 +361,23 @@ func (rc relationsCache) impl(
 	}
 }
 
+type snapshotTestingMatch struct{}
```

Comment:
I'm not sure I understand how this fits together with the PR description. The `Capabilities` function has already existed - it seems like we're adding a capability here? 
If so, why?

Reply:
Okay that makes sense.

I guess to "accidentally" use `snapshot_testing.match` it would still need to get through PR review anyways. And even if one used that function, it's a no-op anyways  😄